third_party/networkpolicy/port_range_test.go

```diff
@@ -0,0 +1,144 @@
+// Copyright 2017 DigitalOcean.
```

Comment:
The tests in this file will never be run, since we use the following command to run unit tests (see Makefile):
```bash
$(GO) test -race -coverprofile=.coverage/coverage-unit.txt -covermode=atomic -cover github.com/vmware-tanzu/antrea/cmd/... github.com/vmware-tanzu/antrea/pkg/...
```

Was that intentional?

Reply:
I have a question about this. This is from the third party lib. Do we need this test file as well?

Reply:
Either make sure the tests are run or remove the file altogether. I think removing the file is fine

Reply:
I will remove it.